frontend/packages/neuroglancer/src/NeuroglancerState.ts

```diff
@@ -0,0 +1,96 @@
+export interface CompleteStateOfANeuroglancerInstance {
```

Comment:
I think it would be nice if this started with a kind of "Do not modify this file by hand. Automatically generated from the neuroglancer viewer state and layer schema via quicktype, run pnpm generate-interface to update this file" 

What do you think?